### PR TITLE
Fix on Programs Grid and Avatar

### PR DESCRIPTION
**Pull request by jagoncalves14**

### CODE REVIEW

File: app/javascript/components/party/index.js [hunk 1]
```diff
@@ -35,14 +35,18 @@ export default class Party extends PureComponent {
     }
 
     componentDidMount() {
+        console.log(this.props.match.params.id)
```
Comment (inesrosete) [inline app/javascript/components/party/index.js]: can be removed

File: app/javascript/components/party/index.js [hunk 2]
```diff
@@ -35,14 +35,18 @@ export default class Party extends PureComponent {
...
         fetch("/parties/" + this.props.match.params.id + ".json")
             .then(res => res.json())
-            .then(data =>
+            .then(data => {
+                console.log(data)
```
Comment (inesrosete) [inline app/javascript/components/party/index.js]: same here

File: app/javascript/components/party/index.js [hunk 3]
```diff
@@ -100,7 +104,7 @@ const analytics = {
 const circles = [
     {
         value: "all",
-        label: "All"
+        label: "Todos"
```
Comment (inesrosete) [inline app/javascript/components/party/index.js:105]: 👍

File: app/javascript/components/party/index.js (hunk 2 above)
Comment (jagoncalves14) [reply app/javascript/components/party/index.js]: Done 👍

File: app/javascript/components/party/index.js (hunk 1 above)
Comment (jagoncalves14) [reply app/javascript/components/party/index.js]: Done 👍

**Review by inesrosete — APPROVED** [review]
LGTM

**Pull request merged**